cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -633,7 +633,7 @@ public static final class Dataset {
     public static final String DATA_STORAGE_SQL_DRIVER_DIRECTORY = "data.storage.sql.jdbc.driver.directory";
     public static final String DATA_STORAGE_SQL_JDBC_DRIVER_NAME = "data.storage.sql.jdbc.driver.name";
 
-    // the jdbc connection related properties should be from cdap-security.xml
+    // the jdbc connection related properties should be from cdap-site.xml
```

Comment:
The username and password are from the cdap-security.xml

Reply:
correct, gonna update